charts/refinery/values.yaml

```
@@ -230,6 +230,10 @@ config:
     # listen for requests for /metrics.
     MetricsListenAddr: 0.0.0.0:9090
 
+  # When RulesConfigMapName is left blank, refinery sets up a ConfigMap with the rules defined below.
+  # The alternative is to pass in your own ConfigMap with the rules defined there. The only caveat is 
+  # you need to use rules.yaml as the key, but the value can be templated out.
+  RulesConfigMapName:
```

Let's use `""` as a default value instead of `null`. It doesn't affect the if condition and looks nicer in the template output.

```suggestion
  RulesConfigMapName: ""
```